assets/keymaps/vim.json

```diff
@@ -321,7 +321,6 @@
     "context": "vim_mode == insert",
     "bindings": {
       // experimental keybinding to toggle helix mode
```

Comment:
nit: remove this comment that was left behind :)

Reply:
Thanks for catching that.